docs/source/tutorials/parsl-rp.ipynb

```diff
@@ -304,7 +297,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.10"
+   "version": "3.11.1"
```

Comment:
should we keep PY version any from 3.7, 3.8, 3.9?

Reply:
Apparently, Jupiter does that automatically when you open the notebook :/